clusterloader2/pkg/test/simple_test_executor.go

```diff
@@ -372,6 +372,9 @@ func isErrsCritical(*errors.ErrorList) bool {
 }
 
 func cleanupResources(ctx Context) {
+	if  !ctx.GetClusterFramework().GetClusterConfig().DeleteAutomanagedNamespaces {
```

Comment:
nit: I can imagine that we will be extending cleanupResources to cleanup other stuff in the future.

So I would prefer being more explicit and move it below in a form:
```
if ctx.....DeleteAutomanagedNamespaces {
  if errList := ctx.....DeleteAutomanagedNamespaces(); ...
    // as is now
   }
}

Reply:
sounds good.

Reply:
made the change.